bgpd/bgp_nht.c

```diff
@@ -702,10 +706,13 @@ static void evaluate_paths(struct bgp_nexthop_cache *bnc)
 
 	if (BGP_DEBUG(nht, NHT)) {
 		char buf[PREFIX2STR_BUFFER];
+		char bnc_buf[BNC_FLAG_DUMP_SIZE];
+
 		bnc_str(bnc, buf, PREFIX2STR_BUFFER);
 		zlog_debug(
-			"NH update for %s(%u)(%s) - flags 0x%x chgflags 0x%x - evaluate paths",
-			buf, bnc->srte_color, bnc->bgp->name_pretty, bnc->flags,
+			"NH update for %s(%u)(%s) - flags %s chgflags 0x%x - evaluate paths",
```

Comment:
nit: no need for space before the '-'

Reply:
same comment it looks like?

Reply:
Actually I was wrong on this one - I should have said unneeded space between the %s and chgflags, but this seems eyewateringly pedantic :)